open_humans/templates/member/my-member-settings.html

```diff
@@ -79,7 +79,7 @@ <h5>Allow other members to contact you?</h5>
     </div>
 
     <div class="col-md-6">
-      <a href="{% url 'account_password' %}?next={% url 'my-member-settings' %}"
+      <a href="{% url 'account_password' %}"
```

Comment:
I don't think this should be removed, it seems to be breaking this:
https://github.com/OpenHumans/open-humans/blob/master/open_humans/member_views.py#L190